notebooks/Testgrid_flakiness_detection.ipynb

```diff
@@ -0,0 +1,2600 @@
+{
```

Comment:
<p>You should make it clear that they are passing the test instead of giving the score. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/37/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

Reply:
<p>Done</p>

Reply:
<p>I meant like adding an additional test condition here like <code>if calc_flakiness_score(test_array_google_1) == 0.0: print("PASSED")</code></p>